modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/TopLevelBuild.java

```diff
@@ -520,6 +537,116 @@ protected Element getBaseBranchDetailsElement() {
 		return baseBranchDetailsElement;
 	}
 
+	protected void getBuildFailureElements(Element detailsElement) {
```

Comment:
Methods that start with "get" should not return void. This type of side-effect-y approach is much harder to follow.